### PR TITLE
Bump chef to 15+ and remove dependency to windows cookbook

### PR DESCRIPTION
**Pull request by Annih**
Chef < 15 are EOL and `windows` cookbook is deprecated.
Chef 15+ provides an `archive_file` resource which can help to replace the former `windows_zipfile`

### CODE REVIEW

File: resources/install.rb
Comment (josqu4red) [reply resources/install.rb:24]: I meant how/why a new bundle of an existing version would exist, SHA would change, etc.

**Review by josqu4red — APPROVED** [review]

**Comment by n8dagr87** [comment]
I have a dependency that is waiting on this pull request - how long before approval turns into a merge? @josqu4red

**Comment by josqu4red** [comment]
@n8dagr87 CI/release is currently broken, we need to pass #44 before, which should be done this week.

**Pull request merged**
